src/i18n/Localizable.xcstrings

```diff
@@ -24747,6 +24876,7 @@
       }
     },
     "Transfers" : {
+      "comment" : "is this one used as verb or noun?",
```

Comment:
Noun => number of transfers

Reply:
changed noun to gerund. is this same meaning for transferred? like 11 / 22 transferred. if not, I'll revert this one to original